mappings/net/minecraft/recipe/RecipeMatcher.mapping

```diff
@@ -1,34 +1,100 @@
 CLASS net/minecraft/unmapped/C_nyagoaau net/minecraft/recipe/RecipeMatcher
 	COMMENT Matching class that matches a recipe to its required resources.
 	COMMENT This specifically does not check patterns (See {@link ShapedRecipe} for that).
+	FIELD f_lzaqyyse amounts Lit/unimi/dsi/fastutil/objects/Reference2IntOpenHashMap;
 	METHOD m_ilmcgbtf clear ()V
 	METHOD m_okheizlq consume (Ljava/lang/Object;I)V
 		COMMENT Consumes a resource from the pool of available items.
-		ARG 2 itemId
-			COMMENT the raw id of the item being consumed
-	METHOD m_wkrwvzbp addInput (Ljava/lang/Object;I)V
+		ARG 1 input
+		ARG 2 amount
+	METHOD m_qzcfdjbc hasAny (Ljava/lang/Object;)Z
+		ARG 1 input
+	METHOD m_sbpnsepr account (Ljava/lang/Object;I)V
```

Comment:
It's not clear at all to me what this does, maybe it's terminology idk about?
Same for methods that call it in `RecipeFinder`.

Reply:
i'll work on some basic docs for that!

Reply:
after considering it i've decided to remove the name in favour of `addInput` -- the original idea was "account for this stack" in matching, but you're right it's not very intuitive